examples/federated_learning/mistnet/coco128/interface.py

```diff
@@ -0,0 +1,126 @@
+
+
+data = {
+    "datasource": "YOLO", 
+    "data_params": "./coco128.yaml", 
+    # Where the dataset is located
+    "data_path": "./data/COCO",
+    "train_path": "./data/COCO/coco128/images/train2017/",
+    "test_path": "./data/COCO/coco128/images/train2017/",
+    # number of training examples
+    "num_train_examples": 128,
+    # number of testing examples
+    "num_test_examples": 128,
+    # number of classes
+    "num_classes": 80,
+    # image size
+    "image_size": 640,
+    "classes":
+    [
+        "person",
+        "bicycle",
+        "car",
+        "motorcycle",
+        "airplane",
+        "bus",
+        "train",
+        "truck",
+        "boat",
+        "traffic light",
+        "fire hydrant",
+        "stop sign",
+        "parking meter",
+        "bench",
+        "bird",
+        "cat",
+        "dog",
+        "horse",
+        "sheep",
+        "cow",
+        "elephant",
+        "bear",
+        "zebra",
+        "giraffe",
+        "backpack",
+        "umbrella",
+        "handbag",
+        "tie",
+        "suitcase",
+        "frisbee",
+        "skis",
+        "snowboard",
+        "sports ball",
+        "kite",
+        "baseball bat",
+        "baseball glove",
+        "skateboard",
+        "surfboard",
+        "tennis racket",
+        "bottle",
+        "wine glass",
+        "cup",
+        "fork",
+        "knife",
+        "spoon",
+        "bowl",
+        "banana",
+        "apple",
+        "sandwich",
+        "orange",
+        "broccoli",
+        "carrot",
+        "hot dog",
+        "pizza",
+        "donut",
+        "cake",
+        "chair",
+        "couch",
+        "potted plant",
+        "bed",
+        "dining table",
+        "toilet",
+        "tv",
+        "laptop",
+        "mouse",
+        "remote",
+        "keyboard",
+        "cell phone",
+        "microwave",
+        "oven",
+        "toaster",
+        "sink",
+        "refrigerator",
+        "book",
+        "clock",
+        "vase",
+        "scissors",
+        "teddy bear",
+        "hair drier",
+        "toothbrush",
+    ], 
+}
+    
+trainer = {
+    "type": "yolov5", 
+    "rounds": 1,
+    "target_accuracy": 0.99,
+    "epochs": 500,
+    "batch_size": 16,
+    "optimizer": SGD,
+    "linear_lr": false,
+    # The machine learning model 
+    "model_name": yolov5,
+    "model_config": "./yolov5s.yaml",
+    "train_params": "./hyp.scratch.yaml"
+}
+    
+aggregation = {
+    "type": "mistnet",
+    "cut_layer": 4,
+    "epsilon": 100
+}
+    
+transmitter = {
+    "address": "127.0.0.1",

```

Comment:
`transmitter` runs as a server in the `other node`, how  do we do this mapping @jaypume

Reply:
Jie Pu has updated it.